jwst/photom/photom.py

```diff
@@ -9,6 +9,7 @@
 from stdatamodels.jwst.datamodels import dqflags
 
 from .. lib.wcs_utils import get_wavelengths
+from . import ifu_mrs
```

Comment:
If you follow the suggestion above to rename `ifu_mrs` to `miri_mrs`, this line will need to change.